misc/l10n_testing.md

```diff
@@ -81,7 +81,7 @@ XML parsing errors (aka "the yellow screen of death") cause Firefox to display a
 
 ### Exposed variables and placeholders
 
-Variables and placeholders become exposed in the user interface when the translated string has a typo within the variable or placeholder. Because it is not exactly the variable that the software or website expected to find, the system simply displays the variable rather than its value. A very common example of this is with the variable &browserShortName. If that variable is typed correctly in the translation, it will return the variable's value (in this case, it's `Firefox`). If it contains a typo, the user will see the variable instead (e.g., `&browsershortname`). These are discoverable in the Mozilla localization dashboards and can be corrected in Pontoon.
+Variables and placeholders become exposed in the user interface when the translated string has a typo within the variable or placeholder. Because it is not exactly the variable that the software or website expected to find, the system simply displays the variable rather than its value. A very common example of this is with the variable &browserShortName. If that variable is typed correctly in the translation, it will return the variable's value (in this case, it's `Firefox`). If it contains a typo, the user will see the variable instead (e.g., ``&browsershortname``). These are discoverable in the Mozilla localization dashboards and can be corrected in Pontoon.
```

Comment:
Sorry, potentially confusing comment from me. I meant that there's another brandShortName before that.

This should look like this

```
Variables and placeholders become exposed in the user interface when the translated string has a typo within the variable or placeholder. Because it is not exactly the variable that the software or website expected to find, the system simply displays the variable rather than its value. A very common example of this is with the variable `&browserShortName`. If that variable is typed correctly in the translation, it will return the variable's value (in this case, it's `Firefox`). If it contains a typo, the user will see the variable instead (e.g., `&browsershortname`). These are discoverable in the Mozilla localization dashboards and can be corrected in Pontoon.
```